quickstarts/microsoft.databricks/databricks-accessconnector/azuredeploy.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accessConnectorName": {
+      "type": "string",
+      "metadata": {
+        "description": "Specify the name of the Azure Databricks AccessConnector."
+      }
+    },
+    "managedIdentityName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the UserAssigned Managed Identity."
+      }
+    },
+     "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "resources": [
+    {
+      "name": "[parameters('managedIdentityName')]",
+      "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+      "apiVersion": "2018-11-30",
+      "location": "[parameters('location')]",
+      "properties": {}
```

Comment:
Do we need this empty properties object?

Reply:
Yes Subbu, else its throwing error that Arm template is missing properties